### PR TITLE
257290 Create Land Parcels GraphQL Resolver

### PR DESCRIPTION
**Pull request by csaltmountain**

### CODE REVIEW

**Review by jack-mccabe-defra — APPROVED** [review]
LGTM

File: app/data-sources/rural-payments-agency-land-api.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { RESTDataSource } from '@apollo/datasource-rest'
+
+export class RuralPaymentsAgencyLandAPI extends RESTDataSource {
+  constructor ({ cache }) {
+    super({ cache })
+  }
```
Comment (dtaylor7) [inline app/data-sources/rural-payments-agency-land-api.js:6]: no need for this

File: app/data-sources/rural-payments-agency-land-api.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  baseURL = process.env.RURAL_PAYMENTS_AGENCY_LAND_API_URL
+
+  async getLandParcelsBySbi (sbi) {
```
Comment (dtaylor7) [inline app/data-sources/rural-payments-agency-land-api.js:10]: doesnt need to be async

File: app/data-sources/rural-payments-agency-land-api.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        request: 'GetFeature',
+        typeNames: 'RPA:LandParcels',
+        cql_filter: `SBI=${sbi}`,
+        srsname: 'EPSG:27700',
+        outputFormat: 'application/json'
+      }
+    })
+  }
+
+  async getLandCoversBySbi (sbi) {
```
Comment (dtaylor7) [inline app/data-sources/rural-payments-agency-land-api.js:23]: doesnt need to be async

File: app/graphql/context.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { RuralPaymentsAgencyLandAPI } from '../data-sources/rural-payments-agency-land-api.js'
+import { apolloServer } from './server.js'
+
+export async function context () {
```
Comment (dtaylor7) [inline app/graphql/context.js:4]: doesnt need to be async

File: app/index.js [hunk 5]
```diff
@@ -21,7 +20,7 @@ const init = async () => {
   console.log('Server running on %s', server.info.uri)
 }
 
-process.on('unhandledRejection', err => {
+process.on('unhandledRejection', (err) => {
   console.log(err)
   process.exit(1)
 })
```
Comment (dtaylor7) [inline app/index.js:26]: kill server on error?

Comment (csaltmountain) [reply app/index.js]: Think this is just a formatting change on my part — have removed the parenthesis.

**Review by dtaylor7 — APPROVED** [review]

**Pull request merged**